src/notification.js

```diff
@@ -16,6 +19,7 @@ export class NotificationElement extends LitElement {
   static properties = {
     config: { state: true },
     urls: { state: true },
+    highest: { state: true },
```

Comment:
A more descriptive name would help clarify how this property is used

```suggestion
    version_highest: { state: true },
```